src/main/java/org/generationcp/middleware/dao/dms/EnvironmentDao.java

```diff
@@ -417,20 +370,20 @@ public Integer getLocationIdByProjectNameAndDescriptionAndProgramUUID(
 		return tiMetadata;
 	}
 
-	public List<Geolocation> getEnvironmentGeolocationsForInstances(final Integer studyId, final List<Integer> instanceNumbers) {
-		List<Geolocation> returnList = new ArrayList<>();
+	public List<ExperimentModel> getEnvironmentsForInstances(final Integer studyId, final List<Integer> instanceNumbers) {
```

Comment:
Same here @darla-leafnode 
should this be renamed to getEnvironmentsByEnvironmentNumber? 
Also this function should be moved to the new DAO. To me is very uncommon to have 2 DAO's for the same entity but I understand that it can get very long. So I dont have a strong position on this, but probably it will be easier to maintain merging both